### DISCUSSION TITLE
revit to bim by iTwin Snapshot

### DISCUSSION THREAD

**Discussion by paprophet** [iTwin/itwinjs-core#1370] [Q&A]
now,i change a revit to bim by  iTwin Snapshot,but
show rectangle in revit but show diamond in imodeljs( or ) 
![imodeljs](https://user-images.githubusercontent.com/18022720/117758967-8c418300-b255-11eb-9574-bebf6452aa02.jpg)
![revit](https://user-images.githubusercontent.com/18022720/117758971-8e0b4680-b255-11eb-9632-b15bcfb1c995.jpg)

**Comment by pmconne** [comment]
That looks to me like a rectangle being viewed at an angle. Are Revit and Desktop Starter using the same view orientation? If so supply .rvt and .bim file please.

**Reply by paprophet** [reply]
i will send the file to you by email ! thank you!

**Reply by pmconne** [reply]
Thanks. The geometry is indeed skewed. I will forward this to the people who work on the Revit connector.

**Reply by pmconne** [reply]
The connector team has identified the problem and is working on a fix.
